ospfd/ospf_vty.c

```diff
@@ -713,19 +713,15 @@ DEFUN (ospf_area_range_not_advertise,
 
 DEFUN (no_ospf_area_range,
        no_ospf_area_range_cmd,
-       "no area <A.B.C.D|(0-4294967295)> range A.B.C.D/M [<cost (0-16777215)|advertise [cost (0-16777215)]|not-advertise>]",
+       "no area <A.B.C.D|(0-4294967295)> range A.B.C.D/M [cost (0-16777215)]",
```

Comment:
We should allow entering a full command.

Reply:
Allow full command but ignore them? And much inconsistencies  on `advertise` and `not-advertise` between ospfv2 and ospfv3.

Reply:
I mean if we do `no area 0 range 192.168.0.0/24 not-advertise` we will have a warning, right? It should be accepted as optional (basically ignored).

Reply:
Wouldn't be easier to just convert to DEFPY and add the `no` form together?

Reply:
So, `area 0 range 192.168.0.0/24 advertise cost 8" and `area 0 range 192.168.0.0/24 cost 8`
which one is right or both? Different between ospfv2 and ospfv3.